src/redux/codes/reducers.js

```diff
@@ -81,6 +81,26 @@ const codes = (state = initialState, action = {}) => {
         errorMessage: errorMessageHandling(payload),
       };
 
+    case constants.UNDEPRECATE_LANGUAGE_CODE_FULFILLED:
+      return {
+        ...state,
+        isPending: false,
+      };
+    case constants.UNDEPRECATE_LANGUAGE_CODE_PENDING:
+      return {
+        ...state,
+        isPending: true,
```

Comment:
Idk, it's just an idea and if it is good practice:

Maybe we could make an object PENDING with the pending values: isPending, isError, errorMessage
And then spread it in every pending call?